pkg/plugins/runtime/gateway/testdata/https/10-gateway-route.yaml

```diff
@@ -110,12 +124,28 @@ Clusters:
           '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
           commonHttpProtocolOptions:
             idleTimeout: 3600s
+            maxConnectionDuration: 0s
+            maxStreamDuration: 0s
           explicitHttpConfig:
             httpProtocolOptions: {}
 Endpoints:
   Resources:
     absent-header-match-327fbb6df03159bd:
       clusterName: absent-header-match-327fbb6df03159bd
+      endpoints:
```

Comment:
Why does this change?

Reply:
In the test there was a list of services used in test. 2 of them were missing so I've added them https://github.com/kumahq/kuma/pull/8920/files/db622c3e63841990aace267734440bb355eb1287#diff-0c14701b8b2b3d398ed8d45398372d61c52a9c4c89203d6e0f77351c83ebdefbR94